### DISCUSSION TITLE
How do i see every users email ID as i'm admin?

### DISCUSSION THREAD

**Discussion by amit1075** [community/community#43472] [New to GitHub]
I need to see which email id used by all users in our organization GitHub account.

**Comment by diva-21** [comment]
Hello @amit1075 ,
Probably you may not see their email ids.
Only if they keep in thier personal profiles,it isnt visible.
Thank you

**Reply by amit1075** [reply]
thanks @diva-21

**Reply by diva-21** [reply]
Do you have further questions so that i can help? Or this did you get the right answer for the above question?
